packages/devops/scripts/deployment/setupGoldMaster.sh

```diff
@@ -51,7 +44,15 @@ mkdir -p /home/ubuntu/.local/share/lpass
 
 # clone our repo
 cd /home/ubuntu
-sudo -Hu ubuntu git clone https://github.com/beyondessential/tupaia.git
+# TODO: Remove this once RN-508 is released
```

Comment:
To actually get this working the `setupGoldMaster.sh` needs to be run via ec2 image builder with this branch checked out. It's a tad awkward, but the plan is to change the branch to `dev` once this code hits `dev`, and then remove entirely once it hits master.

Reply:
Is this just for testing purpose?